packages/nimble-components/src/chip/specs/README.md

```diff
@@ -145,12 +145,18 @@ We will provide styling for the `disabled` attribute state.
 _Consider the accessibility of the component, including:_
 
 - _Keyboard Navigation and Focus_
-    - when the chip component is removable, the remove button will be focusable, otherwise it will not receive focus (following the `nimble-banner` pattern).
+    - When the chip is selectable (`selection-mode="single"`) and removable:
+        - The chip itself is focusable and receives keyboard events
+        - Space/Enter toggles the selected state
+        - Escape removes the chip (emits `remove` event)
```

Comment:
Did you consider whether Escape should also emit the event when the chip is not selectable? I'd advocate for it to work in both cases unless there's a reason not to.